### PR TITLE
Doc: fix external links

### PR DESCRIPTION
**Pull request by bmckercher123**
Fix external links and minor formatting issues in Try out node-to-node encryption doc. 

- Request from Alex P.
- Merge to OS 
- Cherry pick to /master

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 2a163c41f8bb7fc2b68d2f48b6d121c65fb49534

https://deploy-preview-3562--calico.netlify.app

**Comment by lxpollitt** [comment]
Looks like maybe you missed one: " support WireGuard" in the before you begin section.

The formatting of the how to section is messed up.  Looks like the note formatting is corrupt in some way.  You can see this in the netlify preview.  As doing these external links is easy to get wrong, I strongly recommend checking the netlify preview before asking for review, even though in theory the changes are small.

Not really related to fixing the external links, but I think this is not in the best place in the left hand nav.  I would put it directly before "Secure Calico component communications".

File: security/try-node-to-node-encryption.md [hunk 1]
```diff
@@ -1,70 +1,68 @@
...
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between nodes for Calico clusters.
 ---
 
 ### Big picture
 
 Enable WireGuard to secure node-to-node traffic in a {{site.prodname}} cluster.
 
-> **Warning!** Node-to-node encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the Calico Users Slack or GitHub). This feature is currently not supported with overlay networks (IP in IP, VXLAN) due to known issues with NodePort services.
-{: .alert .alert-danger }
+> **Warning!** Node-to-node encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the {{site.prodname}} Users Slack or GitHub). This feature is currently not supported with overlay networks (IP in IP, VXLAN) due to known issues with NodePort services.
```
Comment (lxpollitt) [inline security/try-node-to-node-encryption.md:10]: The change to {{site.prodname}} isn't good in this case. We don't have "Calico Enterprise Users Slack", we only have "Calico Users Slack".

File: security/try-node-to-node-encryption.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Verify the operating system(s) running on the nodes in the cluster {% include op
     ```
    For OpenShift, add the Felix configuration with WireGuard enabled [under custom resources]({{ site.baseurl }}/getting-started/openshift/installation#optionally-provide-additional-configuration).    
 
-   > **Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
+   >**Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
```
Comment (lxpollitt) [inline security/try-node-to-node-encryption.md]: I don't know how sensitive jekyl/liquid is with the markdown here, but if I look at other examples that are rendering correctly, that don't have any spaces before the `>` and they have a space after the `>` before the `**note**`.

i.e. perhaps `  >**Notes**` might need to be `> **Notes**`

**Pull request merged**